pkg/kubelet/images/image_gc_manager.go

```diff
@@ -338,13 +338,29 @@ func (im *realImageGCManager) freeSpace(bytesToFree int64, freeTime time.Time) (
 	im.imageRecordsLock.Lock()
 	defer im.imageRecordsLock.Unlock()
 
+	// Make the ListImages into a map to grab an image by ID
+	allImages, err := im.runtime.ListImages()
+	if err != nil {
+		return 0, err
+	}
+	imagesMap := make(map[string]container.Image, len(allImages))
+	for i := 0; i < len(allImages); i += 1 {
```

Comment:
is there a reason you aren't iterating over the list directly here?

Reply:
I got a different review a while ago that said to preallocate the map, because it helped performance on large lists of images

Reply:
Fortunately that doesn't stop you from iterating - Which is nice for not setting off "this seems weird, lets look closer" alarms in people's heads.

```
for idx, img := range allImages
```